lib/fastlane/plugin/google_sheet_localize/actions/google_sheet_localize_action.rb

```diff
@@ -519,30 +519,40 @@ def self.findArgumentsInText(text)
         result = Array.new
         filtered = self.mapInvalidPlaceholder(text)
 
-        stringIndexes = filtered.scan(/%[0-9]?[s@]/)
-        intIndexes = filtered.scan(/%[0-9]?[d]/)
-        floatIndexes = filtered.scan(/%[0-9]?[f]/)
-        doubleIndexes = filtered.scan(/%[0-9]?ld/)
+        stringIndexes = self.scan_str(filtered, /%[0-9]?[s@]/)
+        intIndexes = self.scan_str(filtered, /%[0-9]?[d]/)
+        floatIndexes = self.scan_str(filtered, /%[0-9]?[f]/)
+        doubleIndexes = self.scan_str(filtered, /%[0-9]?ld/)
 
         if stringIndexes.count > 0
-          result = result.concat(stringIndexes.map { |e| { "index": e, "type": "String" }})
+          result = result.concat(stringIndexes.map { |e| { "index": e[0], "offset": e[1], "type": "String" }})
         end
 
         if intIndexes.count > 0
-          result = result.concat(intIndexes.map { |e| { "index": e, "type": "Int" }})
+          result = result.concat(intIndexes.map { |e| { "index": e[0], "offset": e[1], "type": "Int" }})
         end
 
         if floatIndexes.count > 0
-          result = result.concat(floatIndexes.map { |e| { "index": e, "type": "Float" }})
+          result = result.concat(floatIndexes.map { |e| { "index": e[0], "offset": e[1], "type": "Float" }})
         end
 
         if doubleIndexes.count > 0
-          result = result.concat(doubleIndexes.map { |e| { "index": e, "type": "Double" }})
+          result = result.concat(doubleIndexes.map { |e| { "index": e[0], "offset": e[1], "type": "Double" }})
         end
 
+        result = result.sort_by! { |k| k["offset"] }.reverse!
```

Comment:
```suggestion
        result = result.sort_by! { |k| -k["offset"] }
```

Reply:
I tried that :D it crashed, I have no plan why :D

Reply:
(tests would also be nice, but none are setup for this project)

Reply:
Sure, atm I have my local tests, you can see the fastlane lanes. No Unit tests, but hey, at least I test it :D

Reply:
Aber ja da gibt es sicher Verbesserungspotential